README.md

```diff
@@ -1,35 +1,174 @@
 # FAIR Data Point Client
 
-> This is the client for the [FAIR Data Point](https://github.com/FAIRDataTeam/FAIRDataPoint) providing interface for browsing the data and administration.
+> Client application for the [FAIR Data Point](https://github.com/FAIRDataTeam/FAIRDataPoint) providing a user interface for browsing the metadata and administration.
 
 [![Build Status](https://travis-ci.org/FAIRDataTeam/FAIRDataPoint-client.svg?branch=master)](https://travis-ci.org/FAIRDataTeam/FAIRDataPoint-client.svg?branch=master)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE.md)
 
+
+You can run FAIR Data Point without the client if you need the API only. If you want a user interface for browsing the metadata and administration of the FAIR Data Point metadata and users, you can deploy FAIR Data Point together with this client.
+
+FDP Client works as a proxy before the FAIR Data Point itself. It decides which request should pass through to the FDP (e.g., API calls) and which should be handled by the client (requests from browsers). Therefore, you no longer need to have FAIR Data Point exposed publicly.
+
+
 ## Deployment
 
-FAIR Data Point Client is distributed as a Docker image. It is meant to be run together with the FAIR Data Point. The configuration needed is API url (without the trailing slash) of the FDP server as a `API_URL` ENV variable and host name of FDP server for proxying requests when running the Docker image.
+FAIR Data Point Client is distributed as a Docker image. It runs together with the FAIR Data Point.
 
+**Configuration**
 
-You can use [Docker Compose](https://docs.docker.com/compose/) to run FDP and FDP client together:
+| ENV Variable | Description |
+| ---- | --- |
+| FDP_HOST | A hostname of the FAIR Data Point (within the Docker network). |
+| PUBLIC_PATH | Use only if FDP is not running at the root, you need to specify the URL. For example, if you run FDP at https://example.com/fairdatapoint, PUBLIC_PATH should be `/fairdatapoint`. |
+
+**Example**
+
+Here is an example [Docker Compose](https://docs.docker.com/compose/) configuration to run FDP and FDP client together:
 
 ```yaml
 version: '3'
 services:
     server:
         image: faridata/fairdatapoint
```

Comment:
Typo in the image name, I'm not sure about the actual organisation name on dockerhub.